### ISSUE TITLE
Feature: Do not comment when valid labels are present

### ISSUE THREAD

**Issue by abergs** [jesusvasquez333/verify-pr-label-action#6]
Hi!

First of all, thank you. I just implemented this action and it really help us.
However, on any changes on the PR the action keeps commenting that valid labels are present. I would like it to just approve the change.

E.g: https://github.com/abergs/fido2-net-lib/pull/175

**Comment by NathanielHill** [comment]
@abergs I forked this and removed the comment on success. Unfortunately, you're still going to get notifications for a review approval which is what I was trying to avoid.

**Comment by ssbarnea** [comment]
That comments are pure spam as they stack, rendering the action undesirable for any practical use. I hope this will be addressed soon.

**Comment by NathanielHill** [comment]
@abergs @ssbarnea @mstrk I ended up using this action: https://github.com/mheap/github-action-required-labels

No comments, and the author also optimized it with [`ncc`](https://github.com/vercel/ncc) so it takes about 2 seconds to run!

**Comment by mstrk** [comment]
@NathanielHill thx for the heads-up.

cheers 🖖

**Comment by jesusvasquez333** [comment]
@abergs @NathanielHill @ssbarnea @mstrk Thank you for the feedback. 
I agree that the way it was implemented was very annoying, and it was incorrect to keep commenting the same result. So, I addressed this issue in #12 .

After this change, the action now looks back at the review history and check what was the state of the previous review done by itself (if any), and avoid doing the same action again (for example, if the previous review was approved, and the label are still correct, then the action does not do apply a new review to the PR. Likewise for the opposite case where the last review requested changes and the label are still missing). This will prevent having multiple reviews as @NathanielHill mentioned.

I also removed the `body` of the approval review. Now you will see only one approval notification, without comments in it. I kept whoever the body of the review that request changes.

I just released a new version of this action with this new feature: [v1.2.0](https://github.com/jesusvasquez333/verify-pr-label-action/releases/tag/v1.2.0).

Thanks you all.
